index.html

```diff
@@ -11,10 +12,11 @@
 
       <div class="row form-row">
         <div class="col-md-4 col-md-offset-4">
-          <h1 class="title">Weather App</h1>
+          <h1 class="title">WEATHER TRACKER</h1>
           <form class="getWeatherData">
-            <input type="text" class="weather-city" placeholder="Enter a city.."/>
-            <input type="submit" value="Get Weather">
+            <input type="text" class="weather-city" name="q" placeholder="Enter a city.."/>
+            <input type="hidden" name="appid" value="f44e9d4e81fbd380f38ccb737f016c30">
+            <input type="submit" value="Get Weather" class="submit-button">
             <p class="city-error" style="display:none;">Whoops! Looks like that city doesn't exist, try another one.</p>
```

Comment:
Instead of setting this as an inline style, simply add a "hidden" class and then toggle the class instead of the style with js when you need it to show.